mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/adapters/BankAccountsAdapter.kt

```diff
@@ -0,0 +1,87 @@
+package org.mifos.mobilewallet.mifospay.bank.adapters
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import butterknife.BindView
+import butterknife.ButterKnife
+import org.mifos.mobilewallet.core.domain.model.BankAccountDetails
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.utils.DebugUtil
+import javax.inject.Inject
+
+/**
+ * Created by ankur on 09/July/2018
+ */
+class BankAccountsAdapter @Inject constructor() :
+    RecyclerView.Adapter<BankAccountsAdapter.ViewHolder>() {
+    private var mBankAccountDetailsList: MutableList<BankAccountDetails>?
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val v = LayoutInflater.from(parent.context).inflate(
+            R.layout.item_casual_list,
+            parent, false
+        )
+        return ViewHolder(v)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val bankAccountDetails = mBankAccountDetailsList?.get(position)
+        holder.mTvBankName?.text = bankAccountDetails?.bankName
+        holder.mTvAccountHolderName?.text = bankAccountDetails?.accountholderName
+        holder.mTvBranch?.text = bankAccountDetails?.branch
+        holder.imageViewAccount?.setImageResource(R.drawable.ic_bank)
+    }
+
+    override fun getItemCount(): Int {
+        return mBankAccountDetailsList?.size ?: 0
+    }
+
+    fun setData(bankAccountDetailsList: MutableList<BankAccountDetails>?) {
+        mBankAccountDetailsList = bankAccountDetailsList
+        notifyItemChanged(mBankAccountDetailsList?.size?.minus(1) ?: 0)
```

Comment:
@avinash14022002 this is wrong, `notifyItemChanged` is used to update a single item. Here we are updating the entire list so use `notifyDataSetChanged`